### PR TITLE
feat: bump angular 17

### LINKED ISSUE #87: bump angular 17

### PR DESCRIPTION
**Pull request by damingerdai**
close #87

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,42 +28,42 @@
...
-    "@angular-devkit/schematics": "^16.0.0",
-    "@angular/animations": "^16.0.0",
-    "@angular/common": "^16.0.0",
-    "@angular/compiler": "^16.0.0",
-    "@angular/core": "^16.0.0",
-    "@angular/forms": "^16.0.0",
-    "@angular/platform-browser": "^16.0.0",
-    "@angular/platform-browser-dynamic": "^16.0.0",
-    "@angular/router": "^16.0.0",
+    "@angular-devkit/core": "^17.0.1",
```
Comment (yociduo) [inline package.json:31]: Please using the same version for angular such as the 17.0.0

**Review by yociduo — COMMENTED** [review]
Could you check ci pipeline?

**Comment by yociduo** [comment]
I will remove the node 16 support firstly.

**Comment by yociduo** [comment]
@damingerdai I have removed the node 16 support and fixed the typo for pendo. The CI still failed, please check. By the way, I have rebased your branch, if you do something, please rebase your local branch firstly.

**Comment by yociduo** [comment]
@damingerdai I have removed the node 16 support and fixed the typo for pendo. The CI still failed, please check. By the way, I have cherry pick the remove node 16 support, if you do something, please rebase your branch firstly

**Comment by damingerdai** [comment]
@yociduo please review

**Review by yociduo — APPROVED** [review]

**Pull request merged**